development.md

```diff
@@ -1,152 +1,114 @@
 
+If [NPM](https://www.npmjs.com/get-npm) and [node-gyp](https://github.com/nodejs/node-gyp#installation) don't come bundled with your Node.js installation, you will need to install them manually. 
 
-  **Node JS**
-  Go to the [Node.js Downloads page](https://nodejs.org/en/download/). Download Node.js for MacOS (`v6.9.0` or above recommended). Run the downloaded Node.js `.pkg` Installer. You're finished! To ensure Node.js has been installed, run `node -v` in your terminal - you should get something like `v6.9.0` or above
+### Windows
 
+- Download Node.js for Windows and install it. You can refer to the official docs [here][node-windows] to do so. To ensure Node.js has been installed, run  ```node -v``` in Git Bash to know your node version.
 
-  **if** [**NPM**](https://www.npmjs.com/get-npm) **and** [**node-gyp**](https://github.com/nodejs/node-gyp#installation) **don't come bundled with your Node.js installation, Download manually**
+[node-windows]: https://nodejs.org/en/download/package-manager/#windows
 
+- Also, install install Windows-Build-Tools to compile native node modules by using
+  ```sh
+  $ npm install --global windows-build-tools
+  ```
 
-  Now you are ready for next step [: Get Zulip Desktop Code.](https://github.com/zulip/zulip-desktop/blob/master/development.md#get-zulip-desktop-code)
+## Download, build, and run
 
+Clone the source locally:
+```sh
+$ git clone https://github.com/zulip/zulip-desktop
+$ cd zulip-desktop
+```
 
-## Ubuntu/Debian
+Install project dependencies:
+```sh
+$ npm install
+```
 
+Start the app:
+```sh
+$ npm start
+```
 
-If you’re in a hurry, you can copy and paste the following into your terminal
+Start and watch changes:
+```sh
+$ npm run dev
+```
 
-    sudo apt install git nodejs node-gyp python build-essential snapcraft libxext-dev libxtst-dev lib   xkbfile-dev libgconf-2-4
+Run tests:
+```sh
+$ npm test
+```
 
-after pasting you can jump to next step [: Get Zulip Desktop Code](https://github.com/zulip/zulip-desktop/blob/master/development.md#get-zulip-desktop-code).
+## How to contribute?
 
+Feel free to fork this repository, test it locally and then report any bugs
+you find in the [issue tracker](https://github.com/zulip/zulip-desktop/issues). 
 
-**For a step-by-step explanation, read on.**
+You can read more about making contributions in our [Contributing Guide](./CONTRIBUTING.md).
 
-1. **Node JS**
+## Troubleshooting
 
-	`$ sudo apt-get install nodejs`
+If you have any problems running the app, see the [most common
+issues](./troubleshooting.md).
 
-2. **Install** [**Node-gyp**](https://github.com/nodejs/node-gyp#installation)
+## Making a release
 
-3. **Python (v2.7.x recommended)**
+To package the app into an installer:
+```
+npm run dist
+```
 
-	`$ sudo apt install python2.7`
+This command will produce distributable packages or installers for the
+operating system you're running on:
+* on Windows, a Windows installer file
```

Comment:
We should probably also mention the `nsis.7z` and WebSetup `.exe` file extensions here, because the packaging concept can be a little confusing to new developers.

Reply:
done.